components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/archival/ArchivalService.java

```diff
@@ -0,0 +1,26 @@
+package org.wso2.carbon.device.mgt.core.archival;
```

Comment:
License header should be at the top before the package name declaration.

Reply:
Fixed